CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+## 1.0.0
+
```

Comment:
I think we should write a little line about the fact that the extension is no more a preview and that "development support" by Red Hat is provided.

Reply:
Agree